recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set package_name = "scalene" %}
+{% set version = "1.5.10" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ package_name[0] }}/{{ package_name }}/{{ package_name }}-{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 8e2e2ae14acdca56d5bbf33e158814702410f0a72d4e690fce837b811ffadbfc
+
+build:
+  skip: true  # [py<38]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich >=9.2.0
+    - cloudpickle >=1.5.0
+    - numpy
+    - pynvml >=11.0.0
+
+test:
+  imports:
+    - {{ package_name }}
+  source_files:
+    - test/original/bm_mdp.py
+  requires:
+    - pyperf
+    - pip
+  commands:
+    - pip check
+    - python3 -m scalene --cli test/original/bm_mdp.py  # [not win]
+    - {{ PYTHON }} -m scalene --cli test/original/bm_mdp.py  # [win]
+    
+about:
+  home: https://github.com/plasma-umass/{{ package_name }}
```

Comment:
Using jinja like this obfuscate the recipe, render the links useless when reading/inspecting the recipe only, and doesn't really save us anything. I prefer the actual pkg name here instead.

Reply:
This was a suggested change from a reviewer that I incorporated. It also meets [the DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Anyway, in the interest of getting out of this churn cycle, can we just leave it?

Reply:
Suggested change was here: https://github.com/conda-forge/staged-recipes/pull/18747/commits/2d28eb0cb3e3f7cffde34296e03ebfb78ca1ae46#r886185791

Reply:
OK, will change.